### PR TITLE
Fix CP Filter

### PR DESCRIPTION
**Pull request by jdtoombs**
When 304 was returned sources would consist of an empty array, need to get from local storage when that is the case.

### CODE REVIEW

File: app/subscriber/src/features/home/home-filters/HomeFilters.tsx [hunk 1]
```diff
@@ -18,6 +25,17 @@ export const HomeFilters: React.FC<IHomeFilterProps> = () => {
   const [{ filter }, { storeFilter }] = useContent();
   const [{ sources }] = useSources();
 
+  const [sourcesLookup, setSourcesLookup] = React.useState<ISourceModel[]>([]);
+
+  // if 304 is returned we need to get it from local storage
```
Comment (Fosol) [inline app/subscriber/src/features/home/home-filters/HomeFilters.tsx:30]: I don't understand this.  304 means the content in the endpoint hasn't changed.   Which means you already have the latest content in cache.

You should have a lookup hook like we have for the editor app.  The difference is we don't want all the users being returned in the lookups.

Comment (jdtoombs) [reply app/subscriber/src/features/home/home-filters/HomeFilters.tsx:30]: For some reason it was returning a blank array whenever it was 304, I will try out the lookup hook and maybe update the endpoints and see what happens

**Comment by jdtoombs** [comment]
Having make pack issues....

**Comment by jdtoombs** [comment]
Closing as branch got messy

**Pull request closed without merging**